### PR TITLE
Respect artifact_prefix when writing fuchsia crash artifacts.

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

**Comment by oliverchang** [comment]
TBR. /gcbrun

File: src/python/bot/fuzzers/libfuzzer.py [hunk 1]
```diff
@@ -441,13 +443,15 @@ def fetch_and_process_logs_and_crash(self):
             self.device.fetch(
                 self.fuzzer.data_path(crash_name), self.fuzzer.results_output())
```
Comment (oliverchang) [inline src/python/bot/fuzzers/libfuzzer.py]: @flowerhack We can simplify this by fetching directly into artifact_prefix, but I'm not sure if there are restrictions here in what we can pass as the second argument, so I've done things this way to be safe.

Please change this if we can indeed fetch directly into artifact_prefix

File: src/python/bot/fuzzers/libfuzzer.py [hunk 2]
```diff
@@ -503,14 +507,13 @@ def run_single_testcase(self,
...
 
     # We need to push the testcase to the device and pass in the name.
-    self._test_qemu_ssh()
     testcase_path_name = os.path.basename(os.path.normpath(testcase_path))
     self.device.store(testcase_path, self.fuzzer.data_path())
 
     # TODO(flowerhack): Pass libfuzzer args (additional_args) here
     return_code = self.fuzzer.start(['repro', 'data/' + testcase_path_name])
     self.fuzzer.monitor(return_code)
-    self.fetch_and_process_logs_and_crash()
```
Comment (inferno-chromium) [inline src/python/bot/fuzzers/libfuzzer.py]: Why call this self.process_logs_and_crash() at all if it will bail out instantly on the first line ?

Comment (inferno-chromium) [reply src/python/bot/fuzzers/libfuzzer.py]: If we remove this, then can even remove the artifact_prefix check since we expect it to be always there.

Comment (oliverchang) [reply src/python/bot/fuzzers/libfuzzer.py]: yep I realised this afterwards :) will clean this up

File: src/python/bot/fuzzers/libfuzzer.py (hunk 1 above)
Comment (flowerhack) [reply src/python/bot/fuzzers/libfuzzer.py]: Fetching directly into artifact_prefix should work fine for this function.

I'll put together a follow-up CL shortly to reflect this!

**Pull request merged**